homeassistant/components/wittiot/sensor.py

```diff
@@ -0,0 +1,356 @@
+"""Platform for sensor integration."""
+import dataclasses
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONCENTRATION_PARTS_PER_MILLION,
+    CONF_HOST,
+    DEGREE,
+    PERCENTAGE,
+    UnitOfIrradiance,
+    UnitOfLength,
+    UnitOfPrecipitationDepth,
+    UnitOfPressure,
+    UnitOfSpeed,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import WittiotDataUpdateCoordinator
+
+
+@dataclasses.dataclass(frozen=True)
+class WittiotSensorEntityDescription(SensorEntityDescription):
+    """Class describing WittIOT sensor entities."""
```

Comment:
You're not adding any extra fields so you can just use `SensorEntityDescription`